django/santropolFeast/santropolFeast/templates/base.html

```diff
@@ -23,10 +23,14 @@
             <a href="{% url 'page:login' %}" class="item">Link 1</a>
             <a href="#" class="item">Link 2</a>
             <div class="ui right floated dropdown item">
+                {% if profile.avatar == None %}
                 <img src="{% static 'assets/katrina.jpg' %}" class="ui circular avatar image">
+                {% else %}
+                <img src="/media/{{ profile.avatar }}" class="ui circular avatar image">
+                {% endif %}
                 <div class="content">
                     <div class="ui sub header">{{request.user.username}}</div>
-                    Coordinator
+                    {{profile.description}}
```

Comment:
Description can be 200 character. If you want display the description here you need to split the test with a filter:
https://docs.djangoproject.com/en/1.9/ref/templates/language/#filters